hockeysdk/src/main/java/net/hockeyapp/android/utils/Util.java

```diff
@@ -175,12 +175,7 @@ public static Notification createNotification(Context context, PendingIntent pen
                .setContentText(text)
                .setContentIntent(pendingIntent)
                .setSmallIcon(iconId);
-
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
```

Comment:
Double check that you don't have import that you can remove now that we're not using `Build.*`

Reply:
Now the Build.VERSION.SDK_INT minimum value is 16, and Build.VERSION_CODES.JELLY_BEAN value is 16, so this judgement is always set up. So there is no need judgement, it can be removed.

Reply:
I see it's used elsewhere. I didn't think we had anywhere else that we used `Build`.